packages/wonder-blocks-form/src/components/radio-core.tsx

```diff
@@ -111,7 +106,7 @@ const _generateStyles = (checked: Checked, error: boolean) => {
     const states = {
         // Resting state (unchecked)
         unchecked: {
-            border: semanticColor.border.strong,
+            border: semanticColor.input.default.border,
             background: colorAction.default.background,
```

Comment:
Could this use `semanticColor.input.default.background`? 

On a related note, how do we know to use the input tokens directly vs the secondary action tokens (the `colorAction` variable in this case)?

Reply:
ahh good question... so this is a bit tricky as we need to support default (`blue`) and error (`red`) states. I want to do some styling cleanup here, but I didn't want to do too much just to keep these changes minimal and avoid introducing any visual changes in these components.

I'll also add a TODO comment to come back to this later in https://khanacademy.atlassian.net/browse/WB-1864